py/picca/fitter2/chi2.py

```diff
@@ -294,15 +311,19 @@ def export(self):
         g=f.create_group("best fit")
 
         ## write down all parameters
-        for i, p in enumerate(self.best_fit.values):
+        for i, p in enumerate(self.best_fit.values.to_dict().keys()):
             v = self.best_fit.values[p]
             e = self.best_fit.errors[p]
-            if p in self.best_fit.list_of_fixed_param():
+            if p in [name for (name,fix) in self.best_fit.fixed.to_dict().items() if fix]:
```

Comment:
The list comprehension `[name for (name,fix) in self.best_fit.fixed.to_dict().items() if fix]` appears multiple times in the following lines. Maybe it's good to assign it to a variable before the `for` in lone 314

Reply:
yes, we could. This list-comprehension is what the deprecation warning of list_of_fixed_param() was suggesting to use, so I just used this for the moment (it also ensures we're not accidentally using the list of another fit than the most recent one).

Reply:
ok, I did change to define a variable in the minos and export functions for `fixed_pars` and `varied_pars`, mostly for more verbose code.